OIPA/api/export/tests/test_full_activity.py

```diff
@@ -208,6 +210,7 @@ def test_create_activity(self):
 
 
         xml = iati_activities(
+                ET.Comment(settings.EXPORT_COMMENT),
```

Comment:
@bryanph Just heard it was you who added it, see question above, I'm also interested to know what it does :)